### ISSUE TITLE
sf org list is displaying html for org status when server responds with 503

### ISSUE THREAD

**Issue by peternhale** [forcedotcom/cli#2487]
### Summary
I have an org on an instance and the instance is down for service. The server is responding with a 503. The org list command, in this case, is displaying the raw html response.

### Expected result
org list should not be displaying raw html

### Actual result
```
         xxxxx  foo@some.org            xxxxxxxxxxxxxx <html>                                                                                                                                                                                           
                                                                                    <body>                                                                                                                                                                                           
                                                                                    <center>                                                                                                                                                                                         
                                                                                    	<table bgcolor="white" cellpadding="0" cellspacing="0" width="758">                                                                                                                              
                                                                                    		<tbody>                                                                                                                                                                                          
                                                                                    		<tr>                                                                                                                                                                                             
                                                                                    			<td><span style="font-family: Verdana; font-size: medium; font-weight: bold;">We are down for maintenance.</span><br><br>Sorry for the inconvenience. We'll be back shortly.</td><br>            
                                                                                    		</tr>                                                                                                                                                                                            
                                                                                    		</tbody>                                                                                                                                                                                         
                                                                                    	</table>                                                                                                                                                                                         
                                                                                    </center>                                                                                                                                                                                        
                                                                                    </body>                                                                                                                                                                                          
                                                                                    </html>                                                  

### System Information
zsh
```json
{
  "cliVersion": "@salesforce/cli/2.9.8",
  "architecture": "darwin-x64",
  "nodeVersion": "node-v18.17.1",
  "osVersion": "Darwin 22.6.0",
  "shell": "zsh",
  "rootPath": "/usr/local/lib/node_modules/@salesforce/cli",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 2.3.8 (core)",
    "@oclif/plugin-commands 2.2.25 (core)",
    "@oclif/plugin-help 5.2.19 (core)",
    "@oclif/plugin-not-found 2.4.1 (core)",
    "@oclif/plugin-plugins 3.6.1 (core)",
    "@oclif/plugin-search 0.0.22 (core)",
    "@oclif/plugin-update 3.2.3 (core)",
    "@oclif/plugin-version 1.3.10 (core)",
    "@oclif/plugin-warn-if-update-available 2.1.0 (core)",
    "@oclif/plugin-which 2.2.32 (core)",
    "@salesforce/cli 2.9.8 (core)",
    "apex 2.3.14 (core)",
    "auth 2.8.16 (core)",
    "custom-metadata 2.1.19 (user)",
    "data 2.5.9 (core)",
    "deploy-retrieve 1.17.13 (core)",
    "dev 0.7.13 (user)",
    "functions 1.21.8 (user)",
    "info 2.6.42 (core)",
    "limits 2.3.34 (core)",
    "login 1.2.30 (core)",
    "marketplace 0.2.1 (core)",
    "org 2.10.7 (core)",
    "schema 2.3.25 (core)",
    "settings 1.4.28 (core)",
    "signups 1.4.14 (user)",
    "sobject 0.2.7 (core)",
    "source 2.10.33 (core)",
    "telemetry 2.3.2 (core)",
    "templates 55.5.11 (core)",
    "trust 2.6.10 (core)",
    "user 2.3.32 (core)"
  ]
}
```

**Comment by peternhale** [comment]
The display of raw html is also occurring when running `sf org display`

**Comment by iowillhoit** [comment]
Fixed in the latest version of `sf` `2.12.9`
